ilastik_feature_selection/wrapper_feature_selection.py

```diff
@@ -45,7 +44,7 @@ def kfold_train_and_predict(X, Y, classifier, k=5, indices=None, features=None):
         if features is None:
             features = np.array(list(range(X.shape[1])))
         features = np.array(list(features))
-        kf = cross_validation.KFold(len(indices), n_folds=k)
+        kf = model_selection.KFold(len(indices), n_folds=k)
```

Comment:
the interface changed. now the only parameter you'd give is `n_splits`, which is equivalent to `n_folds`.